pallets/tinlake-investor-pool/src/tests.rs

```diff
@@ -287,14 +291,21 @@ fn epoch() {
 			CurrencyId::Usd,
 			10_000 * CURRENCY
 		));
+		assert_ok!(TinlakeInvestorPool::set_pool_metadata(
+			pool_owner.clone(),
+			0,
+			"QmUTwA6RTUb1FbJCeM1D4G4JaMHAbPehK6WwCfykJixjm3"
```

Comment:
Is it maybe worth leaving a comment explaining how you generated this value?

Reply:
It's just a test IPFS hash, it doesn't really matter what's entered here. I'll add a comment explaining this.